libosmscout/include/osmscout/util/GeoBox.h

```diff
@@ -110,6 +110,25 @@ namespace osmscout {
     }
 
     /**
+     * Create new GeoBox from intersection of this with other
+     * If not Intersects, invalid GeoBox is returned
+     * @param other
+     */
+    GeoBox Intersection(const GeoBox& other) const
```

Comment:
It should be inline or in `GeoBox.cpp`. It seems that import for `std::max` missing here...